### DISCUSSION TITLE
Tkinter Toolkit

### DISCUSSION THREAD

**Discussion by RaKiing45** [Akascape/tkinter-toolkit#6] [Q&A]
Hi, I really wanted to use you Tkinter Toolkit but I have an error with image loading. Is there a fix for it?

**Comment by Akascape** [comment]
@RaKiing45 
Your python version? Please update to latest version if possible

**Reply by RaKiing45** [reply]
I have the newest one

**Reply by Akascape** [reply]
Don't know where the problem is, but it is working fine on my device. win11. Does it shows any error? Can also be related to your internet connection,

**Comment by thisal-d** [comment]
> Hi, I really wanted to use you Tkinter Toolkit but I have an error with image loading. Is there a fix for it?

What's the exception you're getting? Providing an exception will help
